logpyle/__init__.py

```diff
@@ -986,6 +975,8 @@ def save(self) -> None:
         try:
             self.db_conn.commit()
         except OperationalError as e:
+            # Even when encountering a commit error, we want to continue
+            # running the application.
             from warnings import warn
             warn("encountered sqlite error during commit: %s" % e)
```

Comment:
Maybe this should also not be done if the file is not writeable?

Reply:
Agree

Reply:
Done in bcc1001